tracer/src/Datadog.Trace/ClrProfiler/NativeMethods.cs

```diff
@@ -110,6 +110,23 @@ public static void AddDerivedInstrumentations(string id, NativeCallTargetDefinit
             }
         }
 
+        public static void AddInterfaceInstrumentations(string id, NativeCallTargetDefinition[] methodArrays)
+        {
+            if (methodArrays is null || methodArrays.Length == 0)
+            {
+                return;
+            }
+
+            if (IsWindows)
```

Comment:
I’m waiting for 3.0 work to remove this condition on all calls, the pinvokemaps rewriting already solves this.